en/services/gimbal_v2.md

```diff
@@ -0,0 +1,282 @@
+
+<!-- Mermaid graph: 
+graph LR
+	ap["Autopilot"]
+	cc["Companion (Gimbal Manager)"]
+	g["Gimbal Device"]
+	gcs["Ground Station"]
+	ap -- >|GIMBAL_DEVICE_SET_ATTITUDE|g
+	g -- >|GIMBAL_DEVICE_ATTITUDE_STATUS|ap
+	ap -- >|"DO_GIMBAL_MANAGER_ATTITUDE<br/>DO_SET_ROI_LOCATION<br/>(commands in mission)"| cc
+	ap -- >|GIMBAL_DEVICE_ATTITUDE_STATUS|cc
+	gcs -- >|GIMBAL_MANAGER_SET_ATTITUDE|cc
+	cc -- >|GIMBAL_MANAGER_INFORMATION<br/>GIMBAL_MANAGER_STATUS<br/>GIMBAL_DEVICE_ATTITUDE_STATUS|gcs
+	cc -- >|GIMBAL_DEVICE_SET_ATTITUDE|ap
+-->
+
+[![](https://mermaid.ink/img/eyJjb2RlIjoiZ3JhcGggTFJcblx0YXBbXCJBdXRvcGlsb3RcIl1cblx0Y2NbXCJDb21wYW5pb24gKEdpbWJhbCBNYW5hZ2VyKVwiXVxuXHRnW1wiR2ltYmFsIERldmljZVwiXVxuXHRnY3NbXCJHcm91bmQgU3RhdGlvblwiXVxuXHRhcCAtLT58R0lNQkFMX0RFVklDRV9TRVRfQVRUSVRVREV8Z1xuXHRnIC0tPnxHSU1CQUxfREVWSUNFX0FUVElUVURFX1NUQVRVU3xhcFxuXHRhcCAtLT58XCJET19HSU1CQUxfTUFOQUdFUl9BVFRJVFVERTxici8-RE9fU0VUX1JPSV9MT0NBVElPTjxici8-KGNvbW1hbmRzIGluIG1pc3Npb24pXCJ8IGNjXG5cdGFwIC0tPnxHSU1CQUxfREVWSUNFX0FUVElUVURFX1NUQVRVU3xjY1xuXHRnY3MgLS0-fEdJTUJBTF9NQU5BR0VSX1NFVF9BVFRJVFVERXxjY1xuXHRjYyAtLT58R0lNQkFMX01BTkFHRVJfSU5GT1JNQVRJT048YnIvPkdJTUJBTF9NQU5BR0VSX1NUQVRVUzxici8-R0lNQkFMX0RFVklDRV9BVFRJVFVERV9TVEFUVVN8Z2NzXG5cdGNjIC0tPnxHSU1CQUxfREVWSUNFX1NFVF9BVFRJVFVERXxhcCIsIm1lcm1haWQiOnsidGhlbWUiOiJkZWZhdWx0In0sInVwZGF0ZUVkaXRvciI6ZmFsc2V9)](https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZ3JhcGggTFJcblx0YXBbXCJBdXRvcGlsb3RcIl1cblx0Y2NbXCJDb21wYW5pb24gKEdpbWJhbCBNYW5hZ2VyKVwiXVxuXHRnW1wiR2ltYmFsIERldmljZVwiXVxuXHRnY3NbXCJHcm91bmQgU3RhdGlvblwiXVxuXHRhcCAtLT58R0lNQkFMX0RFVklDRV9TRVRfQVRUSVRVREV8Z1xuXHRnIC0tPnxHSU1CQUxfREVWSUNFX0FUVElUVURFX1NUQVRVU3xhcFxuXHRhcCAtLT58XCJET19HSU1CQUxfTUFOQUdFUl9BVFRJVFVERTxici8-RE9fU0VUX1JPSV9MT0NBVElPTjxici8-KGNvbW1hbmRzIGluIG1pc3Npb24pXCJ8IGNjXG5cdGFwIC0tPnxHSU1CQUxfREVWSUNFX0FUVElUVURFX1NUQVRVU3xjY1xuXHRnY3MgLS0-fEdJTUJBTF9NQU5BR0VSX1NFVF9BVFRJVFVERXxjY1xuXHRjYyAtLT58R0lNQkFMX01BTkFHRVJfSU5GT1JNQVRJT048YnIvPkdJTUJBTF9NQU5BR0VSX1NUQVRVUzxici8-R0lNQkFMX0RFVklDRV9BVFRJVFVERV9TVEFUVVN8Z2NzXG5cdGNjIC0tPnxHSU1CQUxfREVWSUNFX1NFVF9BVFRJVFVERXxhcCIsIm1lcm1haWQiOnsidGhlbWUiOiJkZWZhdWx0In0sInVwZGF0ZUVkaXRvciI6ZmFsc2V9)
+
+
+### Message flow
+
+The ground station or any other user facing API such as an SDK are never to send messages directly to the gimbal device but always to the gimbal manager.
+However, the gimbal can and should broadcast its status to everyone, not just the gimbal manager.
+
+> **Note** The gimbal device must act only upon gimbal device messages that come from the gimbal manager!
+
+### Multiple gimbals
+
+Multiple gimbals per drone are supported.
+
+#### Component IDs
+
+Multiple component IDs are reserved for gimbal devices: `MAV_COMP_ID_GIMBAL`, `MAV_COMP_ID_GIMBAL2`, `MAV_COMP_ID_GIMBAL3`, `MAV_COMP_ID_GIMBAL4`, `MAV_COMP_ID_GIMBAL5`, `MAV_COMP_ID_GIMBAL6`.
```

Comment:
I think it might be useful to have a section explaining how the addressing actually works. All the bits are covered here (and can stay here), but pull it out so that it is clear where the messages get sent by the gcs.

Reply:
That's explained right below :confused:.